providers/gce/gce.go

```diff
@@ -96,9 +96,9 @@ var _ cloudprovider.Clusters = (*Cloud)(nil)
 
 type StackType string
 
-const NetworkStackDualStack StackType = "IPV4_IPV6"
-const NetworkStackIPV4 StackType = "IPV4"
-const NetworkStackIPV6 StackType = "IPV6"
+const ClusterStackDualStack StackType = "IPV4_IPV6"
```

Comment:
These things are exported so technically this is a breaking change ... We need to be careful with changing these things lightly

Reply:
Do you suggest to keep the old names? They seem confusing to me. There are no usages of these constants within the library, but indeed something external might depend on them. If this is the case, is it ok to make the change anyway and the users would need to rename the constants while upgrading the cloud-provider library?

Reply:
The `NetworkStack...` naming convention is indeed not precise, as it is not clear if this is for a "VPC Network" or "GKE cluster Network".

We should not break compatibility and it's better to introduce new variables with the same value, mark the current ones as deprecated and add comments describing the deprecation reason.

Reply:
Done.